File: 1.py
#!/usr/bin/env python

# summ = 0
# for i in xrange(334):
# 	summ += i * 3
# 	if (i * 5)%3 != 0 and i * 5 < 1000:
# 		summ += i * 5
# print 'Total sum: {0}'.format(summ)

print 'Total sum:', sum(filter(lambda x: not (x % 3 and x % 5), range(1000)))

print 'Total sum:', sum(x for x in range(1000) if not (x % 3 and x % 5))

File: 1.c
#include <stdio.h>

int main(void) {
	int sum = 0, m, n;

	for (m = 1; m <= 333; ++m) {
		// printf("%i\n", m * 3);
		sum += (m * 3);
	}

	for (n = 1; n <= 199; ++n)
		if ((5 * n) % 3 != 0) {
			sum += (n * 5);
			// printf("%i\n", n * 5);
		}
		
	printf("%i\n", sum);
}
